esrally/track/params.py

```diff
@@ -16,6 +16,7 @@
 # under the License.
 
 import collections
+import inspect
```

Comment:
Is this in here due to a merge?

Reply:
I dont think so, i think its because of collections being added above. Its pushed `inspect` down.